.github/workflows/scripts/test_disable_statements_check.py

```
@@ -0,0 +1,268 @@
+            
+            with patch.object(sys, 'argv', ['disable_statements_check.py', '--files', temp_file]):
+                from disable_statements_check import main
+                # Should return normally (no exception) when no violations
+                try:
+                    main()
+                    # If we reach here, main() returned normally (success)
+                    success = True
+                except SystemExit as e:
+                    # If main() calls sys.exit(), check the code
+                    success = (e.code == 0)
+                self.assertTrue(success)
```

Ruff (0.14.8)

Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)